docs/librarymanager.md

```diff
@@ -0,0 +1,491 @@
+# The Library Manager
+
+Scene Builder is designed to be used with a set of built-in controls, that can be selected from the Library panel (at the left)
+
+<p align="center"><img src="images/librarymanager/SceneBuilder01.png" alt="Controls under Library Panel" width="600"></p>
+
+However, many projects use extra sets of controls, which will be called from now on _custom controls_, that is, JavaFX controls that are not in the built-in set of default controls.
+
+The Library Manager deals with custom controls in different ways, allowing the user not only to import local jar or FXML files, but also to import jars from a number of repositories, remote and local, public and even private ones.
+
+On top of the Library pane item, there is a small MenuButton control with a cog icon, press it to show the Library Manager dialog:
+
+<p align="center"><img src="images/librarymanager/SceneBuilder02.png" alt="Library menuButton" width="400"></p>
+
+<h4>View as List or Sections options</h4>
```

Comment:
replace all `<h4>` with `####` or `###`

Reply:
done

Reply:
After testing locally, I feel `####` or `<h4>` is too small. Can we replace it with `###` instead?

Reply:
done